WebApiCoreSeed.UnitTests/Services/UserServiceTests.cs

```diff
@@ -0,0 +1,52 @@
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using System;
+using System.Threading.Tasks;
+using WebApiCoreSeed.Data.Models;
+using WebApiCoreSeed.Domain.Services;
+using WebApiCoreSeed.Domain.Services.Interfaces;
+
+namespace WebApiCoreSeed.UnitTests.Services
+{
+    [TestClass]
+    public class UserServiceTests : BaseServiceUnitTest
+    {
+        public static User User { get; set; }
+
+        [ClassInitialize]

```

Comment:
It is very arguable, but I thought that current consensus was in the direction of avoiding setup code, inheritance, and class properties on unit tests. This [NUnit post](http://jamesnewkirk.typepad.com/posts/2007/09/why-you-should-.html) is an example.